### ISSUE TITLE
About acc

### ISSUE THREAD

**Issue by sleepy-syr** [ronghuaiyang/arcface-pytorch#79]
Why is acc calculated in this way?acc = np.mean((output == label).astype(int))
The accs obtained by training my own data set are all 0.0625, 0.125, 0.1875 these numbers repeatedly appear,does it have anything to do with the calculation method of acc?And the acc on the verification set I made is always these numbers or 0.0, it is the result of using the pre-trained model, is there any solution? Thank you.

**Comment by Yuz-nuaa** [comment]
Same problem with you. Have you solve this question?

**Comment by qcymkxyc** [comment]
I have same question with you，I thought the cal_accuracy function is incorrect
